modules/clients/src/test/java/org/apache/ignite/qa/query/WarningOnBigQueryResultsTest.java

```diff
@@ -146,13 +154,16 @@ public void testQueryCacheTest0() throws Exception {
      */
     @Test
     public void testQueryInsideCompute() throws Exception {
+        String label = label();
+
         List<List<?>> res = grid("cli").compute(grid("cli").cluster().forNode(grid(0).localNode())).call(
             new IgniteCallable<List<List<?>>>() {
                 @IgniteInstanceResource
                 Ignite ign;
 
                 @Override public List<List<?>> call() throws Exception {
-                    return ign.cache("test0").query(new SqlFieldsQuery("SELECT * FROM TEST0").setLazy(lazy())).getAll();
+                    return ign.cache("test0").query(new SqlFieldsQuery("SELECT * FROM TEST0")
+                        .setLabel(label).setLazy(lazy())).getAll();
```

Comment:
if it supposed to be nullable do we need to label with @nullable appropriate code in IndexQuery ?

Reply:
Added `@nullable` to `IndexQuery`. and `SqlFieldsQuery`